README.md

```diff
@@ -1,33 +1,22 @@
 # screaming-dinosaur
 
+A Rails app for the management of on-call schedules and interactions with Twilio.
+
 ![screaming dinosaur](app/assets/images/screaming_dinosaur.jpg)
 
 [![Build Status](https://travis-ci.org/umts/screaming-dinosaur.svg?branch=master)](https://travis-ci.org/umts/screaming-dinosaur)
 [![Code Climate](https://codeclimate.com/github/umts/screaming-dinosaur/badges/gpa.svg)](https://codeclimate.com/github/umts/screaming-dinosaur)
 [![Test Coverage](https://codeclimate.com/github/umts/screaming-dinosaur/badges/coverage.svg)](https://codeclimate.com/github/umts/screaming-dinosaur/coverage)
 [![Issue Count](https://codeclimate.com/github/umts/screaming-dinosaur/badges/issue_count.svg)](https://codeclimate.com/github/umts/screaming-dinosaur)
 
-Other candidates for the repo name included:
-
-+ rough-morning
-+ persistent-weeknight
-+ digital-moose
-+ nervous-neutron
-
-Rails 5 app for management of Transportation IT on-call schedule and interaction with Twilio.
-
 ## Development
 
 ### Setup
-1. `bundle`
-2. `yarn`
+1. Install Gems: `bundle`
+2. Install Node packages: `yarn`
 3. Create your application.yml: `cp config/application.yml.example config/application.yml`
 4. Create your database.yml: `cp config/database.yml.example config/database.yml`
-5. Create the databases: `rails db:create`
-6. Setup databases: `rails db:schema:load`
-7. Seed development data: `rails db:seed`
-
-When seeding, you can skip creating assignments with `SKIP_ASSIGNMENTS=true rake db:reset` etc.
+5. Setup your databases: `rails db:setup`
```

Comment:
I'll raise you one and point out that this whole section can be replaced with:

```sh
bin/setup
```

(although if we're putting _that_ down maybe now's a good time to move the `setup` and `update` scripts from the `bin/` stub folder to `script/`)

Reply:
Oh neat! That's new to me.

Reply:
If I remember correctly, the main reason would be organization - I think this was a conversation I had with @werebus a long time ago but I'm shakier on it now. As is on round-three, the classifications are "`bin/` for all of rails' binstubs specifically, `script/` for all scriptable development tasks (i.e. `setup`, `update`, `server` when using foreman, etc)".

I'd be open to revisiting that conversation I guess though.

Reply:
In case you haven't already seen it, the [Rails Getting Started Guid](https://guides.rubyonrails.org/getting_started.html#creating-the-blog-application) says: 

> [bin/] Contains the rails script that starts your app and can contain other scripts you use to set up, update, deploy, or run your application.

Reply:
Yeah I'm aware, I think back then it was just a case of us disagreeing with rails' opinion on the matter